data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -9,54 +9,102 @@ fi
 ARTIFACTS="/tmp/artifacts-${GATHER_ID}"
 mkdir -p "${ARTIFACTS}"
 
-exec &> >(tee "${ARTIFACTS}/gather.log")
 
-echo "Gathering bootstrap systemd summary ..."
-LANG=POSIX systemctl list-units --state=failed >& "${ARTIFACTS}/failed-units.txt"
+BOOTSTRAP_IN_PLACE_BOOTSTRAP_PHASE_LOG_BUNDLE_NAME="log-bundle-bootstrap-in-place-pre-reboot"
+BOOTSTRAP_IN_PLACE_BOOTSTRAP_PHASE_ARCHIVE_PATH="/var/log/$BOOTSTRAP_IN_PLACE_BOOTSTRAP_PHASE_LOG_BUNDLE_NAME.tar.gz"
 
-echo "Gathering bootstrap failed systemd unit status ..."
-mkdir -p "${ARTIFACTS}/unit-status"
-sed -n 's/^\* \([^ ]*\) .*/\1/p' < "${ARTIFACTS}/failed-units.txt" | while read -r UNIT
-do
-    systemctl status --full "${UNIT}" >& "${ARTIFACTS}/unit-status/${UNIT}.txt"
-    journalctl -u "${UNIT}" > "${ARTIFACTS}/unit-status/${UNIT}.log"
-done
+if [[ -f ${BOOTSTRAP_IN_PLACE_BOOTSTRAP_PHASE_ARCHIVE_PATH} ]]; then
```

Comment:
Please add a comment here and in the else about when you expect this file to exist

Reply:
b1c9eb40a